Dockerfile

```diff
@@ -4,14 +4,20 @@ USER root
 WORKDIR /cvdi-stream
 
 # Add build tools.
-RUN apt-get update && apt-get install -y g++
+RUN apt-get update && apt-get install -y software-properties-common wget git make gcc-7 g++-7 gcc-7-base && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 100 && update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-7 100
```

Comment:
Are all these additional packages required for some reason now?

Reply:
This is how we were originally installing the build tools.  We're still using this method in the Dockerfile for the ACM so if I had to guess I'd say I made this change to standardize our build tools installation across our C++ projects.

This is the last commit to modify the build tools installation for the ACM:
https://github.com/CDOT-CV/asn1_codec/commit/65f0b546e07c77e85d96c72d457874bfa1404d56